### ISSUE TITLE
Add document title to result download

### ISSUE THREAD

**Issue by lauraykerr** [CDCgov/prime-simplereport#4009]
## Area or Component
Result download 

## Quick Summary
The result download PDF is missing a document title (metadata). 

## Accessibility Violation
PDFs should have a descriptive document title — not just an H1 tag — to give people reading using screen readers proper context. 

## Action Requested
If possible, add 
For context, the way to fix this on Adobe Acrobat is `Document Properties > Description > Title` and `Document Properties > Initial View > Window Options > Show` is set to `Document Title`.

**Comment by emyl3** [comment]
@emmastephenson I believe our current implementation of calling `window.print()` to bring up the system print modal and thus the option to save the result as a pdf doesn't give us the ability to set pdf metadata like the title. I am assuming this story https://app.zenhub.com/workspaces/simplereport-accessibility-61153c23e7546f0010407728/issues/cdcgov/prime-simplereport/4007 may be in a similar problem. 

Do you want me to explore other implementation options so this is possible like looking into libraries that might make this easier? If we want to fix this issue, I'm leaning towards looking at libraries that do this.

**Comment by emyl3** [comment]
Will do -- thank you for your input!!

**Comment by emyl3** [comment]
Used this library https://github.com/diegomura/react-pdf as it had compatibility with react and  lots of activity while also appearing as the go-to library for this kind of effort. It was also not the easiest to style and also threw a bunch of warnings in IntelliJ.

This was able to add the title and language to the pdf. HOWEVER!! It resolved these two issues but introduced like 10 more that were more serious in severity. The effort to use this and the drawbacks of this library was not worth adding this fix. A search of their issues shows their  current accessibility issues: https://github.com/diegomura/react-pdf/search?q=accessibility&type=issues

Closing this ticket.
